lib/google/ads/google_ads/magic_fields.rb

```diff
@@ -0,0 +1,76 @@
+require 'google/protobuf/message_exts'
+require 'google/protobuf/wrappers_pb'
+module Google
+  module Ads
+    module GoogleAds
+      module MagicFields
```

Comment:
Can we call this AutoWrapperFields or AutoWrappingFields or something more else descriptive than just "magic"?